### PR TITLE
Add basic file sharing ui tests

### LINKED ISSUE #1673: add ui test coverage for basic share folder creation
As described, add new test coverage that covers the basic functionality of creating a shared folder and adding files to it.

### PR DESCRIPTION
**Pull request by asnaith**
closes #1673

This is a new test to cover the basic behaviour of creating, updating, editing and deleting a shared folder

### CODE REVIEW

File: packages/files-ui/cypress/tests/file-sharing-spec.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      fileUploadModal.body().should("not.exist")
+      uploadCompleteToast.body().should("be.visible")
+      uploadCompleteToast.closeButton().click()
+      sharedPage.fileItemRow().should("have.length", 1)
+
+      // delete a shared folder
+
+    })
+  })
+})
```
Comment (asnaith) [inline packages/files-ui/cypress/tests/file-sharing-spec.ts:48]: The spec file is not complete atm so please disregard the test steps at the moment as they may not work.

File: packages/files-ui/cypress/tests/file-sharing-spec.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import { createEditSharedFolderModal } from "../support/page-objects/modals/createSharedFolderModal"
+import { fileUploadModal } from "../support/page-objects/modals/fileUploadModal"
+import { navigationMenu } from "../support/page-objects/navigationMenu"
+import { sharingExplainerKey, sharedFolderName } from "../fixtures/filesTestData"
+import { sharedPage } from "../support/page-objects/sharedPage"
+import { uploadCompleteToast } from "../support/page-objects/toasts/uploadCompleteToast"
+
+describe("File Sharing", () => {
```
Comment (asnaith) [inline packages/files-ui/cypress/tests/file-sharing-spec.ts:8]: The spec file is not complete atm so please disregard the test steps at the moment as they may not work.

**Comment by Tbaut** [comment]
The `apiHelper` had a generic function to "clear" buckets, e.g delete any file/folder inside buckets, but it did not handle the bucket deletion (this doesn't make sense for the Home and Bin buckets). So I created a new `deleteSharedBuckets` function that allows to list and delete them all. I also made sure the Share folder overview actually refreshes when it's loaded, so that navigating away and back would show a fresh list of buckets. You should be set now @asnaith

**Comment by asnaith** [comment]
> The `apiHelper` had a generic function to "clear" buckets, e.g delete any file/folder inside buckets, but it did not handle the bucket deletion (this doesn't make sense for the Home and Bin buckets). So I created a new `deleteSharedBuckets` function that allows to list and delete them all. I also made sure the Share folder overview actually refreshes when it's loaded, so that navigating away and back would show a fresh list of buckets. You should be set now @asnaith

This is working perfectly, thank you so much @Tbaut, really appreciate the unblock 🙏

**Review by RyRy79261 — APPROVED** [review]

**Review by Tbaut — APPROVED** [review]
Looks great!

**Pull request merged**